webapp/src/main/java/io/antmedia/enterprise/streamapp/AMSBroadcastManager.java

```diff
@@ -0,0 +1,58 @@
+package io.antmedia.enterprise.streamapp;
+
+import io.antmedia.AntMediaApplicationAdapter;
+import io.antmedia.datastore.db.DataStore;
+import io.antmedia.datastore.db.DataStoreFactory;
+import io.antmedia.datastore.db.IDataStoreFactory;
+import io.antmedia.datastore.db.types.Broadcast;
+import io.antmedia.rest.model.Result;
+import io.antmedia.rest.RestServiceBase;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+
+public class AMSBroadcastManager implements ApplicationContextAware {
```

Comment:
Hi @mustafaboleken ,
What makes you to have another class for this functionality?